src/arrayMethodSort.js

```diff
@@ -3,9 +3,35 @@
 /**
  * Implement method Sort
  */
+const defaultCompare = (previous, current) => {
+  const stringPrevious = String(previous);
+  const stringCurrent = String(current);
+
+  return stringPrevious > stringCurrent
+    ? 1
+    : -1;
```

Comment:
Returning `String(previous) > String(current)` is enough

Reply:
Fixed ✔